awx/settings/defaults.py

```diff
@@ -467,6 +470,10 @@ def IS_TESTING(argv=None):
         'schedule': timedelta(seconds=60),
         'options': {'expires': 50,}
     },
+    'send_subsystem_metrics': {
+        'task': 'awx.main.analytics.analytics_tasks.send_subsystem_metrics',
+        'schedule': timedelta(seconds=3),
```

Comment:
@ryanpetrello this feels unconventional. is using the dispatcher scheduler overkill for a simple 'run this command every 3 seconds" sort of task?

Reply:
Yea, this is probably the wrong way to do this - if the queue gets backed up, this is probably gonna cause it to grow needlessly and run the same thing over and over.

Reply:
If we want to do this ~periodically, perhaps it could be something that happens as part of the callback read loop (maybe we could call this function on every `.read()`, but skip it if N seconds have elapsed?)

cc @chrismeyersfsu for other ideas here